core/src/main/java/org/mskcc/cbio/portal/model/converter/MutationModelConverter.java

```diff
@@ -45,8 +45,8 @@ public ExtendedMutation convert(Mutation mutation) {
         if (mutationEvent.getEndPosition() != null) {
             event.setEndPosition(mutationEvent.getEndPosition());
         }
-        if (mutationEvent.getAminoAcidChange() != null) {
-            event.setProteinChange(mutationEvent.getAminoAcidChange());
+        if (mutationEvent.getProteinChange() != null) {
```

Comment:
Unless the constructor for MutationEvent is doing something unexpected, then the "event" object has all fields starting out as null values. It seems harmless to not test for null return values from getEndPosition(), etc .. just assign the value without a test. If you are assigning a null, you are not changing anything. If these objects are not supposed to have null values in data members, then that's another story --- but if that is true then we *still* should not need to test for null because we would have assured the fields are properly initialized in the constructor and were not null.